src/transformers/modeling_roberta.py

```diff
@@ -139,6 +145,14 @@ def create_position_ids_from_inputs_embeds(self, inputs_embeds):
             Optionally, instead of passing :obj:`input_ids` you can choose to directly pass an embedded representation.
             This is useful if you want more control over how to convert `input_ids` indices into associated vectors
             than the model's internal embedding lookup matrix.
+        encoder_hidden_states  (:obj:`torch.FloatTensor` of shape :obj:`(batch_size, sequence_length, hidden_size)`, `optional`, defaults to :obj:`None`):
```

Comment:
Are those arguments used in any model beside the causal LM one? If not, maybe they should only be exposed/documented there (with `labels`).

Reply:
changed it in BERT as well.